src/metrics.cpp

```diff
@@ -108,6 +108,19 @@ double GetLocalSolPS()
     return miningTimer.rate(solutionTargetChecks);
 }
 
+string WhichNetwork()
+{
+    bool RegTest = GetBoolArg("-regtest", false);
+    bool TestNet = GetBoolArg("-testnet", false);
+    if (TestNet && RegTest)
+        return "";
```

Comment:
This combination will never happen, due to the combination of `NetworkIdFromCommandLine()` and `SelectBaseParams()`. Remove it, and then move the `GetBoolArg()` calls inline below.